generated/backend/FiscalCode.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Do not edit this file it is auto-generated by italia-utils / gen-api-models.
+ * See https://github.com/teamdigitale/italia-utils
+ */
+/* tslint:disable */
+
+import { FiscalCode as FiscalCodeT } from "italia-ts-commons/lib/strings";
```

Comment:
you don't need this generated type, you can use `FiscalCode` imported from `italia-ts-commons/lib/strings`, they are the same type

Reply:
fixed in c5dd8fc